source/operations/checklists/software.rst

```diff
@@ -20,7 +20,8 @@ MinIO Pre-requisites
    :width: 100%
 
    * - :octicon:`circle`
-     - Servers running a Linux operating system with a 5.x+ kernel, such as Red Hat Enterprise Linux (RHEL) 9 or Ubuntu LTS 20.04+
+     - Servers running a Linux operating system with a 5.x+ kernel, such as Red Hat Enterprise Linux (RHEL) 9 or Ubuntu LTS 20.04+.
+       MinIO recommends the latest stable LTS version in the 5.x or 6.x kernel series.
```

Comment:
I'm playing with this in a separate tutorial:


So for second sentence maybe 'Ensure the chosen OS uses LTS and in-support releases of the Linux Kernel 5.x or 6.x series.
```

Reply:
works for me